<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickButtons_ConfigUI</class>
 <widget class="QWidget" name="QuickButtons_ConfigUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>302</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox3">
     <property name="title">
      <string>Available Placeholders</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout">
      <property name="topMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="placeholderHelpLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>%c: Current channel
%i: Input box contents
%k: Channel key
%K: Server password
%n: Send command directly to the server instead of your input box
%o: Current nickname
%s&lt;term&gt;%: term used to separate nicknames in %u
%u: List of selected nicknames</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTreeWidget" name="buttonListView">
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Button Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Button Action</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1" rowspan="4">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QPushButton" name="newButton">
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>230</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Button name:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KLineEdit" name="nameInput">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="actionLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Button action:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KLineEdit" name="actionInput">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="kcfg_ShowQuickButtonsInContextMenu">
     <property name="text">
      <string>Show quick buttons that operate on nicknames in nickname context menus</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonListView</tabstop>
  <tabstop>newButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>nameInput</tabstop>
  <tabstop>actionInput</tabstop>
  <tabstop>kcfg_ShowQuickButtonsInContextMenu</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
